cmd/nerdctl/compose.go

```diff
@@ -82,135 +69,44 @@ func newComposeCommand() *cobra.Command {
 }
 
 func getComposer(cmd *cobra.Command, client *containerd.Client, globalOptions *types.GlobalCommandOptions) (*composer.Composer, error) {
-	nerdctlCmd, nerdctlArgs := globalFlags(cmd)
-	projectDirectory, err := cmd.Flags().GetString("project-directory")
+	options, err := getComposeOptions(cmd)
 	if err != nil {
 		return nil, err
 	}
-	envFile, err := cmd.Flags().GetString("env-file")
+	volStore, err := getVolumeStore(globalOptions)
```

Comment:
You can directly use `volume.Store(globalOptions.Namespace, globalOptions.DataRoot, globalOptions.Address)` in the `pkg` side so there is no need to get `volStore` in `cmd` side and reduce one argument for `compose.GetComposer`).

Below is the definition of `getVolumeStore` and it only wraps a `volume.Store` call. We should remove this wrapper and use `volume.Store` directly. cc @Zheaoli 

https://github.com/containerd/nerdctl/blob/c71c2884a729d3ff8eec0286e39be70c5562f1e4/cmd/nerdctl/volume.go#L47-L49